packages/@sanity/cli/src/actions/init-project/bootstrapTemplate.ts

```diff
@@ -49,6 +55,23 @@ export async function bootstrapTemplate(
     await fs.copyFile(path.join(sharedDir, 'tsconfig.json'), path.join(outputPath, 'tsconfig.json'))
   }
 
+  // If we have a builder ID, the template is assembled from the builder schema
```

Comment:
nit: this comment should probably be updated

Reply:
Updated.